frontend/src/__tests__/basic.test.js

```diff
@@ -0,0 +1,163 @@
+/**
+ * @jest-environment node
+ */
+/* eslint-disable */
+import PropTypes from "prop-types";
+import {
+    apiGET,
+    apiPOST,
+    checkPropTypes,
+    sessionPropTypes,
+    applicantPropTypes,
+    assignmentPropTypes,
+    positionPropTypes
+} from "./utils";
+import { mockAPI } from "../api/mockAPI";
+// eslint-disable-next-line
+const { describe, it, expect, beforeAll, afterAll } = global;
+
+/**
+ * Tests for the API. These are encapsulated in a function so that
+ * different `apiGET` and `apiPOST` functions can be passed in. For example,
+ * they may be functions that make actual requests via http or they may
+ * be from the mock API.
+ *
+ * @param {object} api
+ * @param {Function} api.apiGET A function that when passed a route will return the get response
+ * @param {Function} api.apiPOST A function that when passed a route and data, will return the post response
+ */
+function minimalSetupTest(api = { apiGET, apiPOST }) {
+    const { apiGET, apiPOST } = api;
+    let session = null;
+
+    it("create a session", async () => {
+        await apiPOST("/admin/debug/clear_data");
+
+        // create session
```

Comment:
We need comments explaining what's going on. I.e., that we're seeding, assuming everything is going to work. Then we're testing that things got seeded correctly.